File: python/dummy.py
# dummy.py

def greet(name):
    print(f"Hello, {name}!")

# List of numbers and a simple comprehension
numbers = [1, 2, 3, 4, 5]
squares = [x ** 2 for x in numbers]

if len(numbers) > 3:
    print("More than three numbers.")
else:
    print("Three or less.")

class Person:
    def __init__(self, name, age):
        self.name = name
        self.age = age

    def introduce(self):
        print(f"My name is {self.name} and I am {self.age} years old.")

# Creating an instance of Person
john = Person("John", 30)
john.introduce()

File: ruby/dummy.rb
# dummy.rb

def greet(name)
  puts "Hello, #{name}!"
end

numbers = [1, 2, 3, 4, 5]
squares = numbers.map { |num| num * num }

if numbers.size > 3
  puts "More than three numbers."
else
  puts "Three or less."
end

# Defines a class with some simple methods
class Person
  attr_accessor :name, :age

  def initialize(name, age)
    @name = name
    @age = age
  end

  def introduce
    puts "My name is #{@name} and I am #{@age} years old."
  end
end

# Creating an instance of Person
john = Person.new("John", 30)
john.introduce
